olive/model.py

```diff
@@ -709,13 +724,11 @@ def to_json(self, check_object: bool = False):
 
 
 class OptimumModel(OliveModel):
-    def __init__(self, model_path: str, name: str, model_components: List[str]):
+    def __init__(self, model_path: str, model_components: List[str]):
```

Comment:
we only support string path here. right?

Reply:
good catch! This was added in the middle of my PR so I missed updating here. Done